js/theme.js

```diff
@@ -241,3 +241,37 @@ if (typeof(window) != 'undefined') {
 
 $(".sphx-glr-thumbcontainer").removeAttr("tooltip");
 $("table").removeAttr("border");
+
+function buildSourceType() {
```

Comment:
Since we only use this function once let's not bother with it and just include the code inside the `if` block below